tools/cldr-code/src/main/java/org/unicode/cldr/util/CharUtilities.java

```diff
@@ -102,17 +103,44 @@ public static int compare(CharSource text1, CharSource text2) {
             // if they are different, do a fixup
 
             if (cp1 != cp2) {
-                return (cp1 + utf16Fixup[cp1 >> 11]) -
-                    (cp2 + utf16Fixup[cp2 >> 11]);
+                return (cp1 + utf16Fixup[cp1 >> 11]) - (cp2 + utf16Fixup[cp2 >> 11]);
             }
         }
     }
 
     private static final char utf16Fixup[] = {
-        0, 0, 0, 0, 0, 0, 0, 0,
-        0, 0, 0, 0, 0, 0, 0, 0,
-        0, 0, 0, 0, 0, 0, 0, 0,
-        0, 0, 0, 0x2000, 0xf800, 0xf800, 0xf800, 0xf800
+        0,
```

Comment:
uggg. Any way to manually override (eg with comments) to prevent mangling? We shouldn't have to make code changes just because of a pretty printer!

Reply:
> If I use // @formatter:off and // @formatter:on, it works well.

per https://github.com/jhipster/prettier-java/issues/505#issue-1007366698

Reply:
"// @Formatter:off and // @Formatter:on" -- that looks promising

maybe that wouldn't be needed very often; the utf16Fixup array is unusual due to the performance optimization

Reply:
actually "@Formatter" doesn't work for me, but "// prettier-ignore" does, for java, although the documentation is for javascript:

https://prettier.io/docs/en/ignore.html#javascript